docs/topics/db/examples/proxy_models.txt

```diff
@@ -0,0 +1,96 @@
+===========================
+Proxy model as relationship
+===========================
+
+You can use a proxy model, instead of its concrete model, as ``ForeignKey``,
+``ManyToManyField``, or ``OneToOneField``.
+
+We will use ``Student`` as our main proxy model:
+
+.. code-block:: python
+
+    from django.contrib.auth import get_user_model
+
+    class Student(get_user_model()):
+        class Meta:
+            proxy = True
+
+        def graduate(self):
+            print('Congrats, you got your Pizza degree!')
+
+ForeignKey
+~~~~~~~~~~
+
+.. code-block:: python
+
+    from django.db import models
+
+    class Grade(models.Model):
+        student = models.ForeignKey(Student)
+        value = models.IntegerField()
+
+.. code-block:: pycon
+
+    >>> student = Student.objects.create(first_name='Pizza', last_name='Iolo')
+    >>> grade = Grade.objects.create(student=student, value=480)
+    >>> grade.student.graduate()
+    'Congrats, you got your Pizza degree!'
+    >>> Student.objects.filter(grade__value__gt=450).get()
+    <Student: Pizza Iolo>
+    # You can do the lookup on `Grade` in a `User` queryset
```

Comment:
This kind of comment is what we'd normally put in prose, breaking up the code examples into more manageable steps.